docs/src/apiref.rst

```diff
@@ -68,6 +68,7 @@ Modules:
     models/deprecated/keyedvectors
     models/deprecated/fasttext_wrapper
     models/base_any2vec
+    models/experimental/drmm_tks
```

Comment:
@menshikh-iv 

Please refer to the link below which shows the diff of the requested changes

https://github.com/RaRe-Technologies/gensim/pull/2050/commits/451e3b142fc6cfbd0a0125c66a64257cde7fc92f?utf8=%E2%9C%93&diff=unified

Reply:
Please note

`tox -e docs` will throw errors. Not on my files but on some keras files since I am inheriting from the Keras Layer class which has some unformatted docs.

Reply:
@aneesh-joshi that's shouldn't happen (because you include only your files, not Keras), can you show me log of `tox -e docs` that mention the error in some Keras file (not your)?

Reply:
```
/home/aneeshj/Projects/gensim/.tox/docs/local/lib/python2.7/site-packages/gensim/models/experimental/custom_layers.py:docstring of gensim.models.experimental.custom_layers.TopKLayer.add_weight:10: WARNING: Unexpected indentation.
/home/aneeshj/Projects/gensim/.tox/docs/local/lib/python2.7/site-packages/gensim/models/experimental/custom_layers.py:docstring of gensim.models.experimental.custom_layers.TopKLayer.add_weight:12: WARNING: Block quote ends without a blank line; unexpected unindent.
/home/aneeshj/Projects/gensim/.tox/docs/local/lib/python2.7/site-packages/gensim/models/experimental/custom_layers.py:docstring of gensim.models.experimental.custom_layers.TopKLayer.call:4: WARNING: Inline strong start-string without end-string.
```

I haven't implemented any of the above functions. Just inherited the Layer class.

Reply:
Aham looks like you are right (issue with docstring of the parent class that we can't control).
Simple workaround - define these methods yourself and call super (but don't worry much about it now), you have more critical tasks now.